standalone/rpc-ext/src/lib.rs

```diff
@@ -84,7 +82,7 @@ impl From<Error> for jsonrpc_core::Error {
 
 #[rpc]
 pub trait NodeRpcExtApi<BlockHash> {
-    /// Return the storage changes for each block one by one from `from` to `to` in reversed order.
+    /// Return the storage changes for each block one by one from `from` to `to`(both inclusive).
```

Comment:
What does "inclusive" mean? Say if we have `from = block1, to = block1`, does it return the storage diff between block0 and block 1?

Reply:
> if we have `from = block1, to = block1`, does it return the storage diff between block0 and block 1?
Yes.